src/oc_erchef/rebar.lock

```diff
@@ -17,11 +17,11 @@
   0},
  {<<"chef_authn">>,
   {git,"https://github.com/chef/chef_authn",
```

Comment:
For this PR probably should only update chef_secrets and leave the other ones alone. We're so far behind on things that there's a bit of risk in updating them. Once this merges, I'd suggest we do a *separate* PR that updates the rebar.lock globally and run that through CI.

Reply:
Yup, the blanket dependency update happened accidentally, and the build subsequently failed presumably because of that.  So reverting the change and going with chef_secrets.